docs/hazmat/backends/openssl.rst

```diff
@@ -36,6 +36,19 @@ greater.
 
         The string name of this backend: ``"openssl"``
 
+    .. method:: openssl_version_text()
+
+        :return str: The friendly string name of the loaded OpenSSL library.
+            This is not necessarily the same version as it was compiled against.
+
+    .. method:: openssl_version_number()
+
+        .. versionadded:: 1.8
+
+        :return int: The integer version of the OpenSSL library. This is
```

Comment:
Since you specified that `text` is the linked version, you probably ought to say this is the compiled version not the linked version.